### ISSUE TITLE
Make the share popup window location center on the browser.

### ISSUE THREAD

**Issue by davidrichied** [warfare-plugins/social-warfare#446]
Hi,
I noticed an issue (more of a user experience issue), and I wanted to find out if it's possible to fix/update it.
If you go to a site that is using the social warfare plugin like [https://www.nutsandboltsmedia.com/surviving-client-meetings/](https://www.nutsandboltsmedia.com/surviving-client-meetings/) and click the Facebook share button, the share popup window appears in the very top left of my screen (see image 1).
If you go to a site that uses the AddThis plugin like https://njmonthly.com/articles/eat-drink/a-toast-to-a-healthier-tipple/ and click the Facebook share button (it's fixed to the left of the browser window), the share popup window opens in the center of the screen (see image 2).

TL;DR
Is it possible to open the share popup window in the center of the screen instead of the top left of the screen?

Image 1: [https://ibb.co/eQBxr6](https://ibb.co/eQBxr6)
Image 2: [https://ibb.co/gucyym](https://ibb.co/gucyym)

I emailed Andrea, and she said to submit the issue here.

Thanks in advance!

**Comment by DustinWStout** [comment]
Hmm... very interesting @davidrichied. I think that's definitely worth looking into. I'll leave it to our dev team to investigate further.

**Comment by DustinWStout** [comment]
Oh, and just to clarify, the popup window behavior is 100% powered by the native social network share api. Meaning, this is their script that opens and positions the popup window. However, I think if AddThis has been able to manipulate the positioning, we probably can too (hypothesis).

**Comment by ckmahoney** [comment]
Yes we can! The update has been made and our windows popup windows are happily centered.

Closing on commit `c01045b`.

**Comment by Nicholas-Cardot** [comment]
You need to minify the JS to complete this task.

**Comment by davidrichied** [comment]
That's awesome! Thanks, Cortland! I definitely appreciate it.
